src/Adapter/PlatformServices.Desktop/Services/DesktopTestSourceHost.cs

```diff
@@ -222,10 +132,16 @@ public object CreateInstanceForType(Type type, object[] args)
         /// </summary>
         public void Dispose()
         {
-            if (this.assemblyResolver != null)
+            if (this.parentDomainAssemblyResolver != null)
+            {
+                this.parentDomainAssemblyResolver.Dispose();
```

Comment:
>Dispose [](start = 50, length = 7)

will this throw ?

Reply:
No. Unload() can throw but that is already taken care of.